### ISSUE TITLE
Bitget Bug [safe_string_2()]

### ISSUE THREAD

**Issue by SpenTappy** [ccxt/ccxt#12377]
Hi Kroitor how's it going.

We are using **Bitget** exchange for the first time with a mature road-tested CCXT application.  Some calls to '_safe_string_2_' are missing the non-optional '_key2_' parameter.  It's not immediately obvious which call this is even after stepping through.  

It happens with '_create_order()'_, also fetching open orders.  Note that '_fetchBalance_' works just fine, including calls to _safe_string_2_().  The created orders do seem to be hitting the exchange, I have seen order IDs and balance changes.

The code is mature and works for many exchanges and thousands of orders.  The issue is specific to Bitget.

- OS: MS WIndows Server 2019 Datacenter
- Programming Language version: Python 3.9 (latest)
- CCXT version:  1.75.65 (sorry not able to update on this machine right now but this is fairly up to date version of CCXT)

Instantiation
```
self.exchange_class = getattr(ccxt, exchange_id) 
self.ccxt = self.exchange_class(
  {     'apiKey': config['apiKey'],     
       'secret': config['secret'],     
       'uid': config['uid'],     
       'password': config['password'],     
       'options': {        
               'defaultType': os.getenv('INSTRUMENT'),     
        },     
        'enableRateLimit': True,
    }
) 
 self.ccxt.load_markets()
```

Code:
```
try:     
result = self.ccxt.create_order(         
  order.symbol,         
  order.order_type,         
  order.side,         
  order.amount,         
  order.price,         
  all_params,     
)

Example Values:
BTC/USDT
limit
buy
0.001
42000
{ClientOrderID: 23148132870643187}
```

Error output
```
ERROR | order failed: "safe_string_2() missing 1 required positional argument: 'key2'"
ERROR | Get open orders FAILED for Bitget - safe_string_2() missing 1 required positional argument: 'key2'
```

**Comment by kroitor** [comment]
@SpenTappy hi! That was a typo-bug, thx for pointing it out! I've pushed a fix, it will arrive in CCXT 1.76.36 in ~20 minutes. Please, let us know if that helps or not.

**Comment by SpenTappy** [comment]
Dear @kroitor thank you very much for the rapid intervention.
The fix has solved the problem, orders are now placed and open positions now retrieved in Bitget.
